libs/common/domain/src/lib/record/metadata.model.ts

```diff
@@ -58,10 +58,14 @@ export interface BaseRecord {
   keywords: Array<string> // TODO: handle thesaurus and id
   accessConstraints: Array<AccessConstraint>
   useLimitations: Array<string>
+  legalConstraints?: Array<string>
   licenses: Array<License>
   overviews: Array<GraphicOverview>
   extras?: Record<string, unknown>
   landingPage?: URL
+  topic?: Array<string>
+  updateFrequency?: UpdateFrequency
+  qualityScore?: number
```

Comment:
This should go in `extras` I think

Reply:
I don't understand this comment, can you more details?

Reply:
So the mapper can output things that are too specific for the pivot format in an `extras` field, see for example:
https://github.com/geonetwork/geonetwork-ui/blob/7a84a5b053a3d56ae57f563d8793a40b4cca528f/libs/api/metadata-converter/src/lib/gn4/gn4.field.mapper.ts#L205-L211

I think you should also use `addExtra` for the `qualityScore` field